src/app/myfeed/page.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+import { Grid2 as Grid, Tab, Tabs } from "@mui/material";
+import React, { useState } from "react";
+import PostsFeed from "../_components/myfeed/PostsFeed";
+const MyFeed = () => {
+  const [value, setValue] = useState(0);
```

Comment:
Can't we use enums instead of using index for each tab

Reply:
it seems to be working for integers only